client/users/profile/profile.ng.jade

```diff
@@ -0,0 +1,52 @@
+.profile
+    .form-container(layout="row", layout-sm="column", layout-align="center center")
+        md-whiteframe.md-whiteframe-z1.username-form
```

Comment:
Jak sobie odpaliłem lokalnie to widzę, że tylko input do zmiany loginu ma stałą szerokość, reszta "pływa" jak się na nie kliknie (w sensie placeholdery idą do góry i całość jest troszkę większa). Wg mnie lepiej żeby wszystkie inputy były jak ten z loginu (stała wysokość) bo inaczej cała strona się rusza